### ISSUE TITLE
QA Ph3_Accessibility - Mobile: Home - I/O2016 Twitter tile label is not user friendly with talk back

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#669]
With talk back enabled, the I/O 2016 Twitter tile is labeled as "search q=% I/O 2016" which doesn't exactly make sense, especially if user is unable to see the tile.  

Steps: 
1. Enable talk back in device accessibility settings
2. Go to https://io-webapp-staging.appspot.com/io2016/
3. Scroll down 
4. Tap on the #io2016 tile 
5. Note the label 

Browser/Device: 
Chrome - Google Nexus 6 / Android 5.1.1

![gio_669](https://cloud.githubusercontent.com/assets/15128124/14471240/73c8ffde-00a2-11e6-905b-3b7f694e456b.png)
